neural_compressor/strategy/mse_v2.py

```diff
@@ -24,88 +26,36 @@
-                                               Tuner will combine model, eval_dataloader and
-                                               pre-defined metrics to run evaluation process.
-        eval_func (function, optional):        The evaluation function provided by user.
-                                               This function takes model as parameter, and
-                                               evaluation dataset and metrics should be
-                                               encapsulated in this function implementation and
-                                               outputs a higher-is-better accuracy scalar value.
-
-                                               The pseudo code should be something like:
-
-                                               def eval_func(model):
-                                                    input, label = dataloader()
-                                                    output = model(input)
-                                                    accuracy = metric(output, label)
-                                                    return accuracy
-        dicts (dict, optional):                The dict containing resume information.
-                                               Defaults to None.
-
+    """The `mse_v2` tuning strategy.
+    
+    Note that, only tensorflow framework and pytorch FX backend is currently supported for mse_v2
+    tuning strategy.
     """
-
-    def __init__(self, model, conf, q_dataloader, q_func=None,
-                 eval_dataloader=None, eval_func=None, dicts=None, q_hooks=None):
-        self.ordered_ops = None
-        super(
-            MSE_V2TuneStrategy,
-            self).__init__(
-            model,
-            conf,
-            q_dataloader,
-            q_func,
-            eval_dataloader,
-            eval_func,
-            dicts,
-            q_hooks)
     
     def _tuning_record_msg(self, records):
         records_str_lst = [[str(e) for e in record] for record in records]
         record_msg = '\n'.join(','.join(record) for record in records_str_lst)
         return record_msg
 
     def next_tune_cfg(self):
-        """The generator of yielding next tuning config to traverse by concrete strategies
-           according to last tuning result.
-
+        """Generate and yield the next tuning config with below order.
+        
+           MSE_v2 is a strategy with a two stages fallback and revert fallback.
+           In the fallback stage, it uses multi-batch data to score the op impact
+           and then fallback the op with the highest score util found the quantized
+           model meets accuracy criteria. 
+           In the revert fallback stage, it also scores the impact of fallback OPs
+           in the previous stage and selects the op with the lowest score to revert
+           the fallback until the quantized model not meets accuracy criteria.
+    
```

Comment:
The order of a strategy's `next_tune_cfg` should be in format of an ordered list, like
```
1. The fallback stage: Use multi-batch data to score the op impact and then fallback
    the op with the highest score util found the quantized model meets accuracy
    criteria. 
2. The revert fallback stage: Score the impact of fallback OPs in the previous stage
    and selects the op with the lowest score to revert the fallback until the quantized 
    model not meets accuracy criteria.
```